### PR TITLE
[release-2.4] Updated create namespace to handle retry logic

### PR DESCRIPTION
**Pull request by dislbenn**
Related issue: https://github.com/open-cluster-management/backlog/issues/17503
Signed-off-by: dislbenn <lavontae.bennett@gmail.com>

### CODE REVIEW

**Comment by dislbenn** [comment]
This PR should fix the job issues that are failing above: https://github.com/openshift/release/pull/24427.

**Comment by dislbenn** [comment]
Reopening this PR.

File: .travis.yml [hunk 1]
```diff
@@ -46,12 +46,16 @@ jobs:
     - stage: build
       name: 'Build the image and push it'
       language: node_js
+      node_js:
+        - 10
```
Comment (jlpadilla) [inline .travis.yml:50]: Why 10? The current version is v17, v10 is no longer supported.

**Review by jlpadilla — APPROVED** [review]
/lgtm
/hold

Approving, but left a question about using node v10, which is too old.

**Comment by dislbenn** [comment]
To prevent this PR from blocking our progression from the item referenced, I will merge this PR. The changes have been confirmed to work; however, we will need to prioritize creating an issue to debug the prow jobs that are being executed within the repo.

**Pull request merged**